DSCResources/MSFT_xRemoteFile/MSFT_xRemoteFile.schema.mof

```diff
@@ -7,6 +7,7 @@ class MSFT_xRemoteFile : OMI_BaseResource
     [Write, Description("User agent for the web request.")] String UserAgent;
     [Write, EmbeddedInstance("MSFT_KeyValuePair"), Description("Headers of the web request.")] String Headers[];
     [Write, EmbeddedInstance("MSFT_Credential"), Description("Specifies a user account that has permission to send the request.")] String Credential;
+    [Write, Description("A boolean value to indicate if the destination contents have to be always kept in sync with remote file specified in the source path.\n")] Boolean MatchSource;
```

Comment:
This description isn't quite right, as we don't look at contents of the Destination and Source files (that would require downloading the file every time we call Test-TargetResource, which would remove all the benefits of having a cache).

I'd suggest writing something like: A Boolean value to indicate whether a remote file should be re-downloaded if file in the DestinationPath was modified locally